release/update-repository-version.sh

```diff
@@ -0,0 +1,182 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+readonly script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+readonly script_name="$(basename "${BASH_SOURCE[0]}")"
+
+readonly tmp_dir=$(mktemp -t -d cc-bump.XXXX)
+readonly hub_bin="${tmp_dir}/hub-bin"
+readonly organization="kata-containers"
+PUSH="false"
+
+cleanup (){
+	[ -d "${tmp_dir}" ] && rm -rf "${tmp_dir}"
+}
+
+trap cleanup EXIT
+
+die()
+{
+	msg="$*"
+	echo "ERROR: ${msg}" >&2
+	exit 1
+}
+
+info()
+{
+	msg="$*"
+	echo "INFO: ${msg}" >&2
+}
+
+build_hub() {
+	info "Get hub"
+	git clone --quiet --depth 1 https://github.com/github/hub.git
+	pushd hub >> /dev/null
+	./script/build -o "${hub_bin}"
+	popd >> /dev/null
+}
+
+
+get_changes() {
+	local current_version=$1
+	[ -n "${current_version}" ] || die "current version not provided"
+
+	changes=$(git log --oneline "${current_version}..HEAD")
+	if [ "${changes}" == "" ]; then
+		echo "Version bump no changes"
+		return
+	fi
+
+	# list all PRs merged from $current_version to HEAD
+	git log --merges "${current_version}..HEAD" | awk '/Merge pull/{getline; getline;print }' | while read pr
+	do
+
+		echo "- ${pr}"
+	done
+
+	echo ""
+
+	# list all commits added in this new version.
+	git log --oneline "${current_version}..HEAD" --no-merges
+
+}
+
+generate_commit() {
+	local new_version=$1
+	local current_version=$2
+
+	[ -n "$new_version" ] || die "no new version"
+	[ -n "$current_version" ] || die "no current version"
+
+	printf "release: Kata Containers %s\n\n" "${new_version}"
+
+	get_changes "$current_version"
+
+}
+
+
+bump_repo() {
+	repo=$1
+	new_version=$2
+	[ -n "${repo}" ] || die "repository not provided"
+	[ -n "$new_version" ] || die "no new version"
+	remote_github="git@github.com:${organization}/${repo}.git"
+	info "Update $repo to version $new_version"
+
+	info "remote: ${remote_github}"
+
+	${hub_bin} clone --quiet "${remote_github}"
+
+	pushd "${repo}" >> /dev/null
+
+	info "Forking remote"
+	${hub_bin} fork --remote-name=fork
+	# All repos we build should have a VERSION file
+	[ -f "VERSION" ] || die "VERSION file not found "
+	current_version="$(cat ./VERSION)"
+
+	info "Creating PR message"
+	# find if project has a release notes script
+	release_notes_script="./scripts/release_notes.sh"
+	notes_file=notes.md
+	if [[ -x "${release_notes_script}" ]]
+	then
+		${release_notes_script} "${current_version}" "${new_version}" > "${notes_file}"
+	else
+		cat << EOT > "${notes_file}"
+# Kata Containers ${new_version}
+
+
+$(get_changes "$current_version")
+
+EOT
+	fi
+
+	info "Updating VERSION file"
+	echo "${new_version}" > VERSION
+	if [ -f "configure.ac" ] ; then
+		sed -i -e \
+			"s|\(AC_INIT(\[[-a-zA-Z\_]*\],\s*\[\)${current_version}\(\]\)|\1${new_version}\2|g"\
```

Comment:
we dont have autoconf based projects should we keep it?

Reply:
I'd remove it for now. But we may need to re-add for https://github.com/kata-containers/shim/issues/83.

/cc @sboeuf.